src/sentry/monitors/endpoints/base.py

```diff
@@ -151,3 +142,59 @@ def convert_args(
 
         kwargs.update({"checkin": checkin})
         return args, kwargs
+
+
+class MonitorIngestEndpoint(MonitorEndpoint):
+    """
+    This type of endpont explicitly only allows for DSN and Token / Key based authentication.
+
+    [!!]: These endpoints are legacy and will be replaced by relay based
+          checkin ingestion in the very near future.
+    """
+
+    # TODO(epurkhiser): This should not extend MonitorEndpoint, that base class
+    # is designed for frontend API endpoints
+
+    authentication_classes = (DSNAuthentication, TokenAuthentication, ApiKeyAuthentication)
```

Comment:
We're explicit here. We probably will want to remove `ApiKey` too, since that won't give us organization context and that is problematic.